underlay/src/main/resources/config/service/aouSR2023Q3R1_local.json

```diff
@@ -0,0 +1,27 @@
+{
+  "underlay": "aouSR2023Q3R1",
+  "bigQuery": {
+    "sourceData": {
+      "projectId": "all-of-us-ehr-dev",
+      "datasetId": "SR2023Q3R1",
+      "sqlSubstitutions": {
+        "omopDataset": "all-of-us-ehr-dev.SR2023Q3R1",
+        "staticTablesDataset": "all-of-us-ehr-dev.SR2023Q3R1"
+      }
+    },
+    "indexData": {
+      "projectId": "all-of-us-ehr-dev",
+      "datasetId": "SR2023Q3R1",
+      "tablePrefix": "T"
+    },
+    "queryProjectId": "all-of-us-ehr-dev",
+    "dataLocation": "us-central1"
+  },
+  "dataflow": {
```

Comment:
Is this meant to be an `indexer` config file, instead of a `service` one? Service does not need dataflow config properties.